torch/csrc/jit/runtime/profiling_graph_executor_impl.h

```diff
@@ -7,9 +7,51 @@ namespace jit {
 enum class FusionBehavior { STATIC, DYNAMIC };
 
 using FusionStrategy = std::vector<std::pair<FusionBehavior, size_t>>;
-// E.g. {(Static, 2), (Dynamic, 10)}
-// Fuse with static shapes twice then fallback to 10 dynamic fusions, then stop
-// // compiling new fusion groups
+// FusionStrategy is used to control the type and number of specializations that
+//   can occur during fusion
+//
+// Usage: provide a list of pairs (type, depth) where type is one of "STATIC" or
+//   "DYNAMIC" and depth is an integer.
+//
+// Behavior - static vs dynamic:
+// - in STATIC fusion, fused ops are compiled to have fixed input shapes. The
+//   input shapes are determined based on a number of initial profiling runs.
+//   The shape is determined based on some initial profiling runs. For example,
```

Comment:
Please clarify what shape is referred to here.

Reply:
i think this is used throughout pytorch code base, e.g. 
![image](https://user-images.githubusercontent.com/11477974/151864827-97fd76d6-d31f-4482-9f7b-006faba76505.png)